### DISCUSSION TITLE
stat-computing-r-python/part-wrangling/02-basic-data-vis

### DISCUSSION THREAD

**Discussion by giscus** [srvanderplas/stat-computing-r-python#26] [General]
# stat-computing-r-python/part-wrangling/02-basic-data-vis

https://srvanderplas.github.io/stat-computing-r-python/part-wrangling/02-basic-data-vis.html

**Comment by BabyBorg** [comment]
Seeing if I can comment :)

**Comment by srvanderplas** [comment]
Indeed you can!
